tests/framework-test/iOS/FrameworkTests.cs

```diff
@@ -28,8 +27,10 @@ public class FrameworkTests {
 		public void CFunction ()
 		{
 			Assert.AreEqual (42, CFunctions.theUltimateAnswer (), "a");
+#if !__MACOS__
```

Comment:
Why was it necessary to exclude these two functions?

Reply:
They are inside the static, not dynamic, binaries and `codesign` does not sign them the same way, i.e. no `_CodeSignature` directory, the signature is embedded in the binary. 

While code signing _works_, verification (by us or the OS) fails.